src/java/org/apache/nutch/metadata/Nutch.java

```diff
@@ -96,7 +96,7 @@
 	 * Similar to the -dir command in the bin/nutch script **/
 	public static final String ARG_SEGMENTDIR = "segment_dir";
 	/** Argument key to specify the location of individual segment for the REST endpoints **/
```

Comment:
Either the comment should be updated or else the value should be updated.

Reply:
@lewismc The motivation is the following : 
Part of the endpoints treat segments as an individual Path, part as an array of Paths. Therefore we have an inconsistent usage of the REST API value. The patch allows to use both of them for all endpoints(either array of paths or an individual path, depends on what you sent).

Reply:
@lewismc The motivation is the following :
Part of the endpoints treat segments as an individual Path, part as an array of Paths. Therefore we have an inconsistent usage of the REST API value. The patch allows to use both of them for all endpoints(either array of paths or an individual path, depends on what you sent).